irispy/spectrograph.py

```diff
@@ -447,7 +447,7 @@ def apply_exposure_time_correction(self, undo=False, force=False):
             mask=self.mask, missing_axis=self.missing_axis)
 
 
-def read_iris_spectrograph_level2_fits(filenames, spectral_windows=None):
+def read_iris_spectrograph_level2_fits(filenames, spectral_windows=None, uncertainty=True, memmap=True):
```

Comment:
memmap should probably default to False because of `do_not_scale_image_data`

Reply:
Yes, I think it makes sense to keep the same default behaviour as fits.io.